internal/ingress/controller/config/config.go

```diff
@@ -447,6 +447,9 @@ type Configuration struct {
 	// MIME Types that will be compressed on-the-fly using Brotli module
 	BrotliTypes string `json:"brotli-types,omitempty"`
 
+	// Enables NGINX JS
+	EnableNJS bool `json:"enable-njs,omitempty"`
```

Comment:
Let's not feature flag it now. I think we can just load the module but not do any operation.

It will help me migrating away from some openresty and C modules that are loaded just for very simple operations